packages/docs/src/hooks/useAssets.tsx

```diff
@@ -19,11 +19,6 @@ export function useAssets() {
 
     // Query assets
     queryAssets();
-
-    fuel.on('assets', queryAssets);
```

Comment:
why did we remove this?

Reply:
This file is not used anymore I had delete it